src/css/dracula.css

```diff
@@ -0,0 +1,42 @@
+/*
+
+    Name:       dracula
+    Author:     Michael Kaminsky (http://github.com/mkaminsky11)
+
+    Original dracula color scheme by Zeno Rocha (https://github.com/zenorocha/dracula-theme)
+
+*/
+
+
+.cm-s-dracula.CodeMirror, .cm-s-dracula .CodeMirror-gutters {
```

Comment:
It's the same css as found in CodeMirror but with the background-color set to same as the input field color in Grafana.